new-content-types.md

```diff
@@ -0,0 +1,10 @@
+# adding new content-type versions to lax
+
+- [ ] start a new branch
+- [ ] ensure the api-raml has these new specs present
+- [ ] open src/core/settings.py for editing
+- [ ] change REST_FRAMEWORK, add new content type version
+- [ ] change ALL_SCHEMA_IDX, add new content type version
+- [ ] in `src/publisher/negotiation.py`, change `_dynamic_types` and add new content type version
+- [ ] in `src/publisher/middleware.py`, add any content-type downgrades
+
```

Comment:
This looks handy as a reminder for next time, I know personally after a number of months I forget this amount of detail 👍

Reply:
I often keep checklists like this around. Whether I remember to look here next time is another thing. It also illustrates that there are too many things that need to be updated. I navigate a lot of these older projects by instinct and muscle memory now ;)